docs/source/migration-guides.rst

```diff
@@ -0,0 +1,258 @@
+Migration guides
+================
+
+Migration from Waffle 2.2.0 to Waffle 2.3.0
+-------------------------------------------
+
+**Created monorepo**
```

Comment:
Why use `**` instead of headings like everywhere else? For level 3 you can use `~` https://docutils.sourceforge.io/docs/user/rst/quickstart.html#sections